v2-components/UserBalances/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { Center, Text, Skeleton } from '@chakra-ui/react';
+import Wei, { wei } from '@synthetixio/wei';
+import { SNXIcon, SUSDIcon } from '../Icons';
+
+interface UserBalancesProps {
+  isSnxLoading: boolean;
+  snxBalance: Wei;
+  isSusdLoading: boolean;
+  susdBalance: Wei;
+}
+
+const UserBalances = ({
+  snxBalance = wei(0),
+  susdBalance = wei(0),
+  isSnxLoading = false,
+  isSusdLoading = false,
+}: UserBalancesProps) => {
+  // Add query hook here on integration
+  return (
+    <>
+      <Center
+        borderWidth="1px"
+        borderRadius="4px"
+        borderRightRadius="0px"
+        borderBottomRightRadius="0px"
+        borderRightWidth="0.5px"
+        borderColor="gray.900"
+        py="6px"
+        px="9.5px"
+        height={10}
+        minWidth={110}
+      >
+        <SNXIcon />
+        <Skeleton ml={2} isLoaded={!isSnxLoading}>
+          <Text variant="nav">{snxBalance.toString(2)}</Text>
+        </Skeleton>
+      </Center>
+      <Center
+        borderWidth="1px"
+        borderRadius="4px"
+        borderLeftWidth="0.5px"
```

Comment:
Borders looks a little of with 0.5px. Can we do `borderLeftWidth={0}` here and `borderRightWidth="1px" on the SNX wrapper

Reply:
What browser are you looking at? No issues on chrome/brave. Happy to change but can you send a screenshot

Reply:
Chrome + Storybook 
Before change:
![Screen Shot 2022-08-31 at 3 11 05 pm](https://user-images.githubusercontent.com/5688912/187597572-ebb8258c-d51a-489c-a02a-101580e6a55f.png)
After change:
![Screen Shot 2022-08-31 at 3 11 57 pm](https://user-images.githubusercontent.com/5688912/187597594-fb62970e-6bf2-4bd2-b9e0-db5b58cbe2ae.png)